apps/04TowersOfHanoi.js

```diff
@@ -43,6 +45,7 @@ function checkForWin() {
 
 function towersOfHanoi(startStack, endStack) {
    if (isLegal(startStack, endStack)){
+
      movePiece(startStack, endStack);
      return true;
```

Comment:
Good work on the app! I will have to run this app with you during class with mocha and node, as I am not sure about the above statement. I will give you full credit for now